packages/compass/src/main/application.ts

```diff
@@ -127,9 +127,9 @@ class CompassApplication {
     const home = app.getPath('home');
     const appData = process.env.LOCALAPPDATA || process.env.APPDATA;
     const logDir =
-      process.env.MONGODB_COMPASS_TEST_LOG_DIR || process.platform === 'win32'
+      process.env.MONGODB_COMPASS_TEST_LOG_DIR ?? (process.platform === 'win32'
         ? path.join(appData || home, 'mongodb', 'compass')
-        : path.join(home, '.mongodb', 'compass');
+        : path.join(home, '.mongodb', 'compass'));
```

Comment:
This previously meant that logs from Compass e2e tests would be written to `~/mongodb/compass` on UNIX platforms, even though the intention was to write them to `MONGODB_COMPASS_TEST_LOG_DIR`. This “worked” because we got the actual path to the log file via a webdriver IPC call, rather than checking the contents of `MONGODB_COMPASS_TEST_LOG_DIR` in the e2e tests.